app/utility/api_docs.py

```diff
@@ -0,0 +1,171 @@
+import copy
+from collections import defaultdict
+
+
+class Responses:
+    DEFAULT_RESPONSE = {
```

Comment:
what's the value in overwriting the standard http error code message?

Reply:
This should just be for the documentation -- not the response that is sent.  This is to include some default responses so that they don't need to be explicitly declared on every decorator.